codespell_lib/data/dictionary.txt

```diff
@@ -3776,15 +3813,26 @@ apostrafies->apostrophes
 apostrafy->apostrophe
 apostrophie->apostrophe
 apostrophies->apostrophes
+appache->apache
 appar->appear
+apparance->appearance
+apparances->appearances
 apparant->apparent
 apparantly->apparently
+appareance->appearance
+appareances->appearances
 appared->appeared
+apparen->apparent
```

Comment:
Perhaps `apparel` too? **L** and **N** are close on Dvorak keyboards, for what it's worth.

Reply:
I couldn't find `apparen->apparel`.